iron-request.html

```diff
@@ -248,7 +249,13 @@
         'xml': 'application/xml',
         'arraybuffer': 'application/octet-stream'
       }[options.handleAs];
-      var headers = options.headers || {};
+      var headers = options.headers || Object.create(null);
```

Comment:
Is `Object.create(null)` preferable to object literal (`{}`) for a particular reason?